doc/api/cluster.markdown

```diff
@@ -302,30 +643,30 @@ because of exiting or being signaled). Otherwise, it returns `false`.
 * `signal` {String} Name of the kill signal to send to the worker
   process.
 
-This function will kill the worker. In the master, it does this by disconnecting
-the `worker.process`, and once disconnected, killing with `signal`. In the
-worker, it does it by disconnecting the channel, and then exiting with code `0`.
+This function will, by default, signal the worker process to terminate. In the
+master process, this is done by first disconnecting the worker process, then
+sending the `signal` (which defaults to `SIGTERM`). Within the worker, the IPC
+channel is disconnected and the process exits with code `0`.
 
-Causes `.suicide` to be set.
+When called, the `worker.suicide` property will be set to `true`.
 
 This method is aliased as `worker.destroy()` for backwards compatibility.
 
-Note that in a worker, `process.kill()` exists, but it is not this function,
-it is [`kill`][].
+Note that in a worker process, the `worker.kill()` and `process.kill()` methods
+are *not* equivalent to one another.
 
 ### worker.process
 
 * {ChildProcess object}
 
-All workers are created using [`child_process.fork()`][], the returned object
-from this function is stored as `.process`. In a worker, the global `process`
-is stored.
+Within the master process, `worker.process` returns the [`ChildProcess`][]
+object returned by [`child_process.fork()`] when the worker was spawned.
 
-See: [Child Process module][]
+Within the worker process, `worker.process` returns the global `process`.
 
 Note that workers will call `process.exit(0)` if the `'disconnect'` event occurs
-on `process` and `.suicide` is not `true`. This protects against accidental
-disconnection.
+on `process` and `worker.suicide` is not `true`. This protects against
```

Comment:
unless I misread the diff, this now says "Note that workers will call `process.exit(0)` if the `'disconnect'` event occurs on `process` and `worker.suicide` is not `true`.", which doesn't make any sense.